.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,36 @@
+## Type
+
+* ### Feature
+  <!--Implements a new feature-->
+* ### Fix
+  <!--Fixes a bug-->
+* ### Docs
+  <!--Documentation only changes-->
+* ### Style
+  <!--Changes that **do not** affect the meaning of the code (white-space, formatting, missing semi-colons, etc)-->
+* ### Refactor
+  <!--A code change that neither fixes a bug nor adds a feature-->
+* ### Performance
+  <!--A code change that improves performance-->
+* ### Test
+  <!--Adding missing or correcting existing tests-->
+* ### Chore
+  <!--Changes to the build process or auxiliary tools and libraries such as documentation generation-->
+
+## Description
+
+<!--A brief description of changes. Things to include: WIP? Dependent PR's opened against other tickets?-->
+
+## Issues
+
+<!--A list of Jira issues closed by this PR.-->
+
+ * Fixes [FRONT-]
```

Comment:
@shaunmaharaj not sure if `FRONT-` is still correct? (this syntax should trigger JIRA github bot to link the PR back to the ticket) `MT-` now maybe?

Reply:
Thanks for catching this @samblacklock, I missed that. Updated now & pushed 👍